basic-example/docker-compose.yml

```diff
@@ -32,13 +32,11 @@ services:
       # All containers will join this network
       DOCKER_NETWORK_NAME: jupyterhub-network
       # JupyterHub will spawn this Notebook image for users
-      DOCKER_NOTEBOOK_IMAGE: jupyter/minimal-notebook:latest
+      DOCKER_NOTEBOOK_IMAGE: jupyter/base-notebook:latest
```

Comment:
`jupyter/base-notebook` is smaller and is a base image, so it's better candidate for testing.